torch/_decomp/decompositions.py

```diff
@@ -1730,3 +1730,70 @@ def nll_loss_forward(
             result = result.sum() / total_weight
 
     return result, total_weight
+
+
+@register_decomposition(aten.mv)
+def mv(self, vec):
+    utils.check(
```

Comment:
should we move this to refs? Also, you should check dtypes, `mv` doesn't accept inputs with non-fp dtypes, and different dtypes, whereas multiplication would

Reply:
lol tests are failing for integral types, because of course
```
AssertionError: Object comparison failed: torch.uint8 != torch.int64 : Operation: aten.mv.default, orig.dtype: torch.uint8, decomp.dtype: torch.int64, (tensor([[5, 1, 6, 5, 6, 4, 2, 5, 5, 9],
        [3, 1, 4, 2, 3, 2, 6, 8, 2, 2],
        [8, 2, 0, 4, 9, 2, 1, 9, 2, 2],
        [9, 4, 4, 7, 2, 5, 8, 6, 6, 7],
        [1, 5, 2, 1, 7, 4, 3, 0, 7, 6]], dtype=torch.uint8), tensor([9, 3, 3, 9, 0, 1, 2, 9, 7, 8], dtype=torch.uint8)), {}
 ```